src/FileService.Business/Commands/Image/FindImagesCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿using LT.DigitalOffice.FileService.Business.Commands.Image.Interfaces;
+using LT.DigitalOffice.FileService.Data.Interfaces;
+using LT.DigitalOffice.FileService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.FileService.Models.Dto.Models;
+using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Responses;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.FileService.Business.Commands.Image
+{
+    public class FindImagesCommand : IFindImagesCommand
+    {
+        private readonly IImageRepository _repository;
+        private readonly IImageInfoMapper _mapper;
+
+        public FindImagesCommand(
+            IImageRepository repository,
+            IImageInfoMapper mapper)
+        {
+            _repository = repository;
+            _mapper = mapper;
+        }
+
+        public OperationResultResponse<List<ImageInfo>> Execute(List<Guid> imageIds)
```

Comment:
Why not FinedResultResponse?

Reply:
this find working without any pagination, so i don't need property total count